pvAccessJava/src/org/epics/ca/ClientFactory.java

```diff
@@ -148,7 +149,7 @@ public void destroy() {
                 context.destroy();
             } catch (CAException e) {
                 e.printStackTrace();
-            }
+        }
```

Comment:
Is this a tab vs. spaces issue?

Reply:
Ops, I missed that. Reverting to the previous state.